### PR TITLE
[Minor][Doc][SQL] Fix two documents regarding size in bytes

### PR DESCRIPTION
**Pull request by viirya**
## What changes were proposed in this pull request?

Fix two places in SQLConf documents regarding size in bytes and statistics.
## How was this patch tested?

No. Just change document.

### CODE REVIEW

**Comment by viirya** [comment]
cc @cloud-fan @liancheng Just minor document changes. Please review if this change is proper. Thanks.

**Comment by cloud-fan** [comment]
LGTM

**Comment by viirya** [comment]
ping @cloud-fan

File: sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala [hunk 1]
```diff
@@ -122,10 +124,10 @@ object SQLConf {
 
   val DEFAULT_SIZE_IN_BYTES = SQLConfigBuilder("spark.sql.defaultSizeInBytes")
     .internal()
-    .doc("The default table size used in query planning. By default, it is set to a larger " +
-      "value than `spark.sql.autoBroadcastJoinThreshold` to be more conservative. That is to say " +
-      "by default the optimizer will not choose to broadcast a table unless it knows for sure " +
-      "its size is small enough.")
+    .doc("The default table size used in query planning. By default, it is set to Long.MaxValue " +
+      "which is more than `spark.sql.autoBroadcastJoinThreshold` to be more conservative. " +
```
Comment (cloud-fan) [inline sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala]: `which is larger than`

Comment (viirya) [inline sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala]: fixed.

**Comment by cloud-fan** [comment]
thanks, merging to master

**Pull request closed without merging**